service/src/main/java/org/sunbird/actor/user/UserBaseActor.java

```diff
@@ -38,6 +38,8 @@
 
   protected UserLookupService userLookupService = UserLookUpServiceImpl.getInstance();
   protected LocationClient locationClient = LocationClientImpl.getInstance();
+  protected UserCreateRequestValidator userCreateRequestValidator =
```

Comment:
if all method inside is static then no need to create object for this class

Reply:
done